hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -74,75 +54,65 @@ function ResetOVSService() {
 }
 
-function RemoveHiddenNetDevices() {
-    $Devs = $(Get-PnpDevice -Class net | ? Status -eq Unknown | Select InstanceId)
-    foreach ($Dev in $Devs) {
-        $RemoveKey = "HKLM:\SYSTEM\CurrentControlSet\Enum\$($Dev.InstanceId)"
-        Get-Item $RemoveKey | Select-Object -ExpandProperty Property | %{Remove-ItemProperty -Path $RemoveKey -Name $_ -Verbose }
-    }
+function CreateStartOVSvSwitchd() {
+    sc.exe create ovs-vswitchd binpath="$OVSInstallDir\usr\sbin\ovs-vswitchd.exe  --pidfile -vfile:info --log-file  --service" start= auto depend= "ovsdb-server"
+    sc.exe failure ovs-vswitchd reset= 0 actions= restart/0/restart/0/restart/0
+    start-service ovs-vswitchd
+    $OVS_VERSION=$(Get-Item $OVSInstallDir\driver\OVSExt.sys).VersionInfo.ProductVersion
+    ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
 }
 
 function clearOVSBridge() {
-    $ovsStatus = $(Get-Service ovs-vswitchd).Status
-    if ("$ovsStatus" -EQ "running") {
-        Write-Host "Delete OVS bridge: br-int"
-        ovs-vsctl.exe --no-wait --if-exists del-br br-int
-        $MaxRetryCount = 10
-        $RetryCountRange = 1..$MaxRetryCount
-        $BrIntDeleted = $false
-        foreach ($RetryCount in $RetryCountRange) {
-            Write-Host "Waiting for OVS bridge deletion complete ($RetryCount/$MaxRetryCount)..."
-            $BrIntAdapter = $(Get-NetAdapter "$OVS_BR_ADAPTER" -ErrorAction SilentlyContinue)
-            if ($BrIntAdapter -eq $null) {
-                $BrIntDeleted = $true
-                break
-            }
-            if ($RetryCount -eq $MaxRetryCount) {
-                break
-            }
-            Start-Sleep -Seconds 5
-        }
-        if (!$BrIntDeleted) {
-            Write-Host "Failed to delete OVS Bridge, please retry the script or delete the bridge and HNS network manually."
-            return
-        }
+    $ovsdbService =  Get-Service ovsdb-server -ErrorAction SilentlyContinue
+    if ( $ovsdbService -ne $null -and $ovsdbService.Status -eq "running") {
+        Write-Host "Delete OVS bridge from OVSDB: br-int"
+        ovs-vsctl.exe --no-wait --if-exists del-br $OVS_BR_ADAPTER
     }
 }
 
-$BrIntDeleted = $(Get-NetAdapter "$OVS_BR_ADAPTER") -Eq $null
-if ($BrIntDeleted -eq $false) {
-    clearOVSBridge
-}
-$uplink = ""
-$AntreaHnsNetworkName = "antrea-hnsnetwork"
-$NetId = GetHnsnetworkId($AntreaHnsNetworkName)
-if ($NetId -ne $null) {
-    Write-Host "Remove HnsNetwork: $AntreaHnsNetworkName"
-    $uplink = $(Get-HnsNetwork -Id $NetId).NetworkAdapterName
-    Get-HnsNetwork -Id $NetId | Remove-HnsNetwork
+function ClearHnsNetwork() {
+    $vmSwitch = Get-VMSwitch -Name $AntreaHnsNetworkName -ErrorAction SilentlyContinue
+    if ($vmSwitch -ne $null) {
+        Write-Host "Remove vNICs"
+        Remove-VMNetworkAdapter -SwitchName $AntreaHnsNetworkName -ManagementOS -Confirm:$false -ErrorAction SilentlyContinue
+        $hnsNetwork = Get-HnsNetwork | ? Name -eq $AntreaHnsNetworkName
```

Comment:
I notice that we use a slightly different command in another script:
```
hack/windows/Prepare-AntreaAgent.ps1:$AntreaHnsNetwork = Get-HnsNetwork | Where-Object {$_.Name -eq "antrea-hnsnetwork"}
```

Reply:
The two commands are the same, `?` is an alias of `Where-Object`. 
I would update to keep consistent.